### PR TITLE
Side 1996 sandbox capability ldk setup

### PR DESCRIPTION
**Pull request by cshields-oliveai**

### CODE REVIEW

File: .github/workflows/goja-node.js.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm set audit false
+      - run: npm ci
+      - run: npm audit --production --audit-level=low
+      - run: npm run build --if-present
+      - run: npm run lint
+      - run: npm run test
+      # - run: npm run doc
```
Comment (cshields-oliveai) [inline .github/workflows/goja-node.js.yml:37]: TODO: Add documentation?

File: ldk/goja/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import * as clipboard from './clipboard';
+
+export { clipboard };
```
Comment (cshields-oliveai) [inline ldk/goja/src/index.ts:3]: TODO: Clipboard will need comments, and we need clipboard lock, listener cancellation which will be resolved in SIDE-2006.

**Review by RusGraf — APPROVED** [review]

File: .github/workflows/goja-node.js.yml (hunk 1 above)
Comment (richseviora) [reply .github/workflows/goja-node.js.yml:37]: I've been thinking about this; I think we will probably want to generate documentation again with Typedoc to the LDK/go folder in the GH pages repo. There's aspects of the current Typedoc presentation I don't love but we don't need to solve that right now.

File: ldk/goja/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: [
+    '@typescript-eslint',
+  ],
+  rules: {
+    'class-methods-use-this': 'off',
+    'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+    'import/prefer-default-export': 'off', // Default exports are bad.
```
Comment (richseviora) [inline ldk/goja/.eslintrc.js:22]: they really bad

File: ldk/goja/src/index.ts (hunk 2 above)
Comment (richseviora) [reply ldk/goja/src/index.ts:3]: This was a placeholder, but now I'm thinking that we should expose an instance of the `ClipboardImpl` class.

**Review by richseviora — CHANGES_REQUESTED** [review]
Minor change that we could make but isn't required yet for this initial work

File: .github/workflows/goja-node.js.yml (hunk 1 above)
Comment (cshields-oliveai) [reply .github/workflows/goja-node.js.yml:37]: I will create a card for this.

Comment (cshields-oliveai) [reply .github/workflows/goja-node.js.yml:37]: https://crosschx.atlassian.net/browse/SIDE-2043

**Review by richseviora — APPROVED** [review]

**Pull request merged**